config/crd/bases/core.openfeature.dev_featureflagconfigurations.yaml

```diff
@@ -39,8 +39,38 @@ spec:
             properties:
               featureFlagSpec:
                 description: FeatureFlagSpec is the json representation of the feature
-                  flag
+                  flag specification
                 type: string
+              featureFlagSpecV2:
```

Comment:
It doesn't look like v2 supports `$evaluators`. Could you please make sure that's supported?

Reply:
I think you're right, good catch: https://github.com/open-feature/flagd/blob/019ab1de02c587c189ca71b68999cb1de44d88f6/config/samples/example_flags.json

Reply:
One last thought on this. Would it be possible to bump the API version from `core.openfeature.dev/v1alpha1` to `core.openfeature.dev/v1alpha2` or `core.openfeature.dev/v1beta1` and use that as an opportunity to update the `featureFlagSpec` property? That may be a clean way to upgrade the CRD while maintaining backwards compatibility.